docs/reference/cli-reference/cli-ref-sources.md

```diff
@@ -53,6 +53,11 @@ where `<operation>` is one of *List, Add, Remove, Enable, Disable,* or *Update*,
 
   Specifies the password for authenticating with the source.
 
+  > [!Note]
+  > Make sure to add the sources' password under the same user context as the nuget.exe is later used to access the package source.
+  > The password will be stored encrypted in the config file and can only be decrypted in the same user context as it was encrypted.
+  > So for example when you use a build server to restore NuGet packages the password must be encrypted with the same Windows user under which the build server task will run.
```

Comment:
Encrypted passwords only work on the same machine, in addition to the same account. Therefore, committing encrypted passwords into source control, or otherwise sharing a nuget.config with encrypted password, never makes sense.

I think the wording you used elsewhere in the PR was better. I'm not sure why build servers are being called out specifically here.

Reply:
> I'm not sure why build servers are being called out specifically here.

I copied the entire paragraph from the existing documentation https://learn.microsoft.com/nuget/reference/cli-reference/cli-ref-sources (refer to `Note` in the doc). I will change this note to the following.

```
    > [!NOTE]
    > Be aware that encrypted passwords are only supported on Windows. 
    > Moreover, they can only be decrypted on the same machine and by the same user who originally encrypted them.
````